src/ir_Hitachi.cpp

```diff
@@ -1785,18 +1790,13 @@ void IRHitachiAc296::send(const uint16_t repeat) {
 }
 #endif  // SEND_HITACHI_AC296
 
-
 /// Get the value of the current power setting.
 /// @return true, the setting is on. false, the setting is off.
-bool IRHitachiAc296::getPower(void) const {
-  return _.Power;
-}
+bool IRHitachiAc296::getPower(void) const { return _.Power; }
```

Comment:
Are these manually or auto-formatted? If auto, would love to know what you've got set up to do that.

Reply:
Alas, manual.

Reply:
😭